### DISCUSSION TITLE
How to style UL child of a Div? (the UL is loaded from database)

### DISCUSSION THREAD

**Discussion by elfeffe** [tailwindlabs/tailwindcss#1909] [Help] [answered]
I need to style some child elements, they are being loaded from a database, can I declare on the DIV that it's children UL's have a list-disc class?

**Comment by royvanv** [comment, marked as answer]
Assuming you are using a custom build, you can use the following code:

```CSS
.parent li {
    @apply pl-4 text-orange-500; /* Or whatever classes you want to use. */
}
```

**Reply by robbyrice** [reply]
Another option would be to add the `list-disc` and `list-inside` classes to each `<ul>` element.

You might want to take a look at these links.
https://tailwindcss.com/docs/list-style-type
https://tailwindcss.com/docs/list-style-position

**Reply by royvanv** [reply]
That would meaning some string replacements on the data after it is fetched from the database.

**Reply by robbyrice** [reply]
I guess I misunderstood the original question. Am I correct in understanding that the whole `<ul>` (meaning all the HTML, not just the data) is being loaded from the database? If that is the case, then yes it would be better to do it in the stylesheet.

```css
.parent ul {
  @apply list-disc list-inside;
}
```

Otherwise, it might be better to do it inside the loop that is generating the HTML.

**Reply by elfeffe** [reply]
Thank you, that was what I was looking for

**Reply by robbyrice** [reply]
Glad to help!
